ocaml-lsp-server/src/ocamlformat_rpc.ml

```diff
@@ -46,15 +47,18 @@ end = struct
       | Ok s -> s
       | Error e -> Exn_with_backtrace.reraise e)
 
-  let configure ~logger { io_thread; client; _ } =
+  let configure ~logger ~format_style { io_thread; client; _ } =
     (* We ask for 64 columns formatting as this appear to be the maximum size of
        VScode popups. TODO We should probably allow some flexibility for other
        editors that use the server. *)
+    let options =
+      match format_style with
+      | `VSCodePopup -> [ ("module-item-spacing", "compact"); ("margin", "63") ]
```

Comment:
Are we pushing vscode limitations onto other editors? Again that seems very wrong.